changelog.d/133.misc

```diff
@@ -0,0 +1 @@
+Don't try an aioapns connection 4 times per APNS dispatch attempt; do the same as GCM.
```

Comment:
Maybe:

> Attempt the same number of retries for both GCM and APNS.

Reply:
How dare you suggest that we write something non-cryptic in a changelog? :)

I agree, will change this over, I must have had some kind of commit writer's block at the time :S!

Reply:
It's hard to write it from the POV of someone running sydent instead of what the internal changes mean!